core/src/Platforms/iOS/EmbeddedWebview/WKWebNavigationDelegate.cs

```diff
@@ -73,7 +73,8 @@ public override void DecidePolicy(WKWebView webView, WKNavigationAction navigati
                    requestUrlString.StartsWith(BrokerConstants.BrowserExtInstallPrefix, StringComparison.OrdinalIgnoreCase))
             {
                 AuthenticationAgentUIViewController.callbackMethod(new AuthorizationResult(AuthorizationStatus.Success, requestUrlString));
-                AuthenticationAgentUIViewController.DismissViewController(true, null);
+                AuthenticationAgentUIViewController.DismissViewController(true, () =>
+                    AuthenticationAgentUIViewController.callbackMethod(new AuthorizationResult(AuthorizationStatus.Success, requestUrlString)));
                 decisionHandler(WKNavigationActionPolicy.Cancel);
```

Comment:
>Cancel [](start = 57, length = 6)

Should this be cancel and not Ok/Success or something similar?

Reply:
Success is what we've had, so I don't think we should change it. we can discuss off-line though.